nc_time_axis/__init__.py

```diff
@@ -49,7 +54,7 @@ def __repr__(self):
 
 class NetCDFTimeDateFormatter(mticker.Formatter):
     """
-    Formatter for cftime.datetime data.
+    Formatter for cftime datetime data.
```

Comment:
Can this have consistent formatting with the [CalendarDateTime docstring](https://github.com/SciTools/nc-time-axis/pull/49/files#diff-250670b4e3f86a94858f6fdc62f01aceR34-R37)?